### PR TITLE
Add one test for sequence type indirection #879

### PR DESCRIPTION
**Pull request by xor2003**
Add one test for sequence type indirection. Relate to issue #879
https://github.com/DOCGroup/ACE_TAO/issues/879

### CODE REVIEW

File: TAO/tests/Any/Indirected/client.cpp [hunk 1]
```diff
@@ -226,6 +228,21 @@ recursive_union_test ()
...
+
+  ACE_DEBUG ((LM_INFO,
+              "* Indirected sequnce demarshaling\n"));
+  {
+  TAO_InputCDR strm((const char*)a_raw, a_raw_len, 0, 1, 2); // big endian, IIOP 1.2
+
+   Test::StructuredEvent notifications;
+   if ((strm >> notifications) == 0)
+   {
+      ACE_DEBUG ((LM_INFO,
```
Comment (jwillemsen) [inline TAO/tests/Any/Indirected/client.cpp]: Should be ERROR, not DEBUG/INFO

File: TAO/tests/Any/Indirected/test.idl [hunk 2]
```diff
@@ -15,4 +15,42 @@ module Test
           RecursiveUnion3 uni;
   };
 
+// Sequnce indireciton test
```
Comment (jwillemsen) [inline TAO/tests/Any/Indirected/test.idl]: Typo indirection

**Comment by xor2003** [comment]
Is it intended behaviour what demarshaling code does not have debug tracing? Is it for performance purpose?

**Comment by xor2003** [comment]
CodeFactor points to code which is not related to PR

**Pull request closed without merging**